three_ANA_sites/siteInfo.csv

```diff
@@ -1,10 +1,10 @@
-station,station_name,latitude,longitude,drainage_area,constituent,constit_units
-RONC02800,Ronc,1,1,100,NO3,mg L^-1
-RONC02800,Ronc,1,1,100,PT,mg L^-1
-RONC02800,Ronc,1,1,100,Q,cms
-MOGU02900,Mogu,2,2,200,NO3,mg L^-1
-MOGU02900,Mogu,2,2,200,PT,mg L^-1
-MOGU02900,Mogu,2,2,200,Q,cms
-ORIZ02900,Oriz,3,3,300,NO3,mg L^-1
-ORIZ02900,Oriz,3,3,300,PT,mg L^-1
-ORIZ02900,Oriz,3,3,300,Q,cms
\ No newline at end of file
+file,site.id,site.name,lat,lon,basin.area,constituent,units
```

Comment:
to your first comment: sure, i'm OK with matchingSite and will rename it. The column already contains the site ID.

to your second comment; it would be nice to distinguish between these units and basin.area.units, yes...but which units are these current ones? they can be either conc.units or flow.units already. conc.or.flow.units would be accurate but seems like a bit much.

i like that this table structure saves some typing by having separate rows for discharge datasets, but there are other options we could consider:

1. Completely flat: exactly one row per run. e.g. these column names:
```
site.id,site.name,lat.lon.basin.area,flow.site.id,flow.site.name,flow.lat.flow.lon.flow.basin.area,basin.area.units,constituent,consti.name,conc.units,flow.units,load.units,...
```
2. More relational: separate files for sites and sets of data inputs
```
siteInfo.csv
---------
site.id,site.name,lat.lon.basin.area,basin.area.units

dataInfo.csv
--------
site.id,constituent,conc.units, flow.site.id,flow,flow.units, load.units,load.rate.units,dates ...
```